system/views/admin/formbuilder/cloneForm.cfm

```diff
@@ -0,0 +1,28 @@
+<cfscript>
+	var id = rc.id ?: "";
+	var formId = createUUID();
```

Comment:
This has the potential to create invalid HTML element IDs (they must not start with a number). I think ok to use "formbuilder-clone-form" or somesuch here.